src/main/java/com/tabgok/heimdallr/Heimdallr.java

```diff
@@ -0,0 +1,22 @@
+package com.tabgok.heimdallr;
+
+import com.tabgok.entity.factory.MachineFactory;
+import com.tabgok.server.Gateway;
+
+
+public class Heimdallr {
```

Comment:
"Heimdallr" is fine but typically naming convention would be to name this "Main" or something descriptive like "HeimdallrServer"